### PR TITLE
Skip call to coreos-setgoodroot if an update has already been completed

### PR DESCRIPTION
**Pull request by honeyankit**
…EED_REBOOT then update-engine.service should not update the priority flags.

### CODE REVIEW

**Comment by marineam** [comment]
test this please

File: src/update_engine/update_attempter.cc [hunk 1]
```diff
@@ -115,8 +115,10 @@ UpdateAttempter::UpdateAttempter(SystemState* system_state,
       start_action_processor_(false) {
   prefs_ = system_state->prefs();
   omaha_request_params_ = system_state->request_params();
-  if (utils::FileExists(kUpdateCompletedMarker))
+  if (utils::FileExists(kUpdateCompletedMarker)){
     status_ = UPDATE_STATUS_UPDATED_NEED_REBOOT;
+    updated_boot_flags_(true);
```
Comment (marineam) [inline src/update_engine/update_attempter.cc]: Use `updated_boot_flags_ = true` instead please.

**Comment by marineam** [comment]
Looks correct, just a couple syntax nits.

File: src/update_engine/update_attempter.cc [hunk 2]
```diff
@@ -115,8 +115,10 @@ UpdateAttempter::UpdateAttempter(SystemState* system_state,
       start_action_processor_(false) {
   prefs_ = system_state->prefs();
   omaha_request_params_ = system_state->request_params();
-  if (utils::FileExists(kUpdateCompletedMarker))
+  if (utils::FileExists(kUpdateCompletedMarker)){
```
Comment (marineam) [inline src/update_engine/update_attempter.cc]:
```
/mnt/host/source/update_engine/source/src/update_engine/update_attempter.cc:120:29: error: expression cannot be used as a function
     updated_boot_flags_(true);
```

**Comment by honeyankit** [comment]
I have made the necessary changes as per @marineam 
Please review it.
Thank you.

**Comment by marineam** [comment]
Thanks! I've verified the change and it works for me.

**Pull request merged**